test/evs/snapshot_test.go

```diff
@@ -0,0 +1,111 @@
+package evs
+
+import (
+	"fmt"
+	"github.com/container-storage-interface/spec/lib/go/csi"
+	"github.com/huaweicloud/huaweicloud-csi-driver/pkg/evs"
+	"github.com/huaweicloud/huaweicloud-csi-driver/pkg/utils"
+	acceptance "github.com/huaweicloud/huaweicloud-csi-driver/test"
+	"testing"
+	"time"
+)
+
+var (
```

Comment:
use const

Reply:
func operations in the code block, can not change to const